src/applications/lgy/coe/containers/introduction-content/COEStatuses/COEPending.jsx

```diff
@@ -5,15 +5,15 @@ const COEPending = props => {
   return (
     <>
       <va-alert status="warning">
-        <h3 slot="headline">
-          {props.pending
+        <h2 slot="headline">
```

Comment:
I think this still might need the class `vads-u-font-size--h3` to maintain parity with the mockups.

Reply:
Interestingly it doesn't, the stying comes from the alert component itself

Reply:
Oh cool! Didn't know that.